app/models/test_group.py

```diff
@@ -90,6 +90,7 @@ def __init__(self, name, lab_name):
         self.time = -1
         self.vcs_commit = None
         self.warnings = 0
+        self.lava_multinode = {}
```

Comment:
The test groups are LAVA agnostic, so leaking LAVA multinode things here is not going to work.

Reply:
I didn't get what you mean, I tested with a multinode job and a none multinode job and there is no error.

Reply:
I mean that this data model should not assume anything about LAVA.  So if you need to add some extra data to each test group or test case, you can do that but don't make it follow the LAVA multinode design.  It needs to be something that works on its own in a self-contained way.

Reply:
So what we need is an abstraction that's not LAVA-centric to describe a group of jobs that are all connected/related.

It appears that the LAVA multinode data has a key called `target_group` which is a unique ID (generated by LAVA) that is shared across all sub-jobs of a multi-node job.

For starters, this would be the minimum thing needed to know that a group of (sub)jobs were all part of the same main (multinode) job.